notebook/static/base/js/dialog.js

```diff
@@ -234,7 +234,8 @@ define(['jquery',
         });
 
         modal_obj.on('shown.bs.modal', function(){ editor.refresh(); });
-        modal_obj.on('hide.bs.modal', function(){options.edit_metadata_button.focus(); });
+        modal_obj.on('hide.bs.modal', function(){
+            options.edit_metadata_button ? options.edit_metadata_button.focus() : ""});
```

Comment:
Should there be a `;` inside the anonymous function?

(Nitpicking,  I know)

Reply:
Done